Apps/Playground/package.json

```diff
@@ -19,9 +19,9 @@
     "@react-native-community/slider": "4.0.0-rc.2",
     "logkitty": "^0.7.1",
     "react": "^17.0.1",
-    "react-native": "0.64.0-rc.2",
+    "react-native": "^0.64.0-rc.0",
```

Comment:
This older version is new enough?

Reply:
If your app has already installed react-native@0.64.0-rc.0 and that's what is in the package.json.lock file, then it won't automatically update to something newer, so does @babylonjs/react-native-windows actually work with 0.64.0-rc.0?

Reply:
I don't think it does. Would peer dependency resolution be able to pick up these items? Seems like maybe I should specify a greater than clause as well